sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -6379,6 +6428,32 @@ object functions {
   def to_json(e: Column): Column =
     to_json(e, Map.empty[String, String])
 
+  // scalastyle:off line.size.limit
+  /**
+   * Masks the given string value. This can be useful for creating copies of tables with sensitive
+   * information removed.
+   *
+   * @param input string value to mask. Supported types: STRING, VARCHAR, CHAR
+   * @param upperChar character to replace upper-case characters with. Specify NULL to retain original character.
+   * @param lowerChar character to replace lower-case characters with. Specify NULL to retain original character.
+   * @param digitChar character to replace digit characters with. Specify NULL to retain original character.
+   * @param otherChar character to replace all other characters with. Specify NULL to retain original character.
+   *
+   * @group string_funcs
+   * @since 3.5.0
+   */
+  // scalastyle:on line.size.limit
+  def mask(
+    input: Column,
+    upperChar: Column,
+    lowerChar: Column,
+    digitChar: Column,
+    otherChar: Column): Column = {
```

Comment:
Please supplement the API with other constructor of `Mask`.

Reply:
Sure, will add the other constuctors. Thanks.